### DISCUSSION TITLE
How do I remove warning colors for tabs and spaces at the end of the line

### DISCUSSION THREAD

**Discussion by datwaft** [wuelnerdotexe/vim-enfocado#7] [Q&A] [answered]
See the following screenshot:

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/37723586/184556507-271cad21-fe71-48a8-a85b-f6bedc5be9dd.png">

How do I remove the color yellow that highlights tabs and spaces at the end of the line?

**Comment by datwaft** [comment, marked as answer]
This seems to be caused by:
https://github.com/wuelnerdotexe/vim-enfocado/blob/2eccaa4e4761a3a8971f4d52b27fd304010896a6/colors/enfocado.vim#L980-L985

It seems that those lines always override:

https://github.com/wuelnerdotexe/vim-enfocado/blob/2eccaa4e4761a3a8971f4d52b27fd304010896a6/colors/enfocado.vim#L241

Even if I don't have the plugin installed.

This seems to happen because `enfocado#pluginIsActivated('whitespace', 0)` always returns `1`, even if I don't have the plugin installed.

Testing it a little bit more, it seems that `enfocado#pluginIsActivated` always returns `1`, even if I pass the name of a plugin that doesn't exist.

As this seems to be a bug I will create an issue from this discussion.

At the moment I added [this](https://github.com/datwaft/nvim.conf/commit/a8b30761319e457e044181bfea31265b7f88da97) workaround to my configuration.

**Reply by wuelnerdotexe** [reply]
Hi David! - in the last update where I added support for the [vim-better-whitespace](https://github.com/ntpeters/vim-better-whitespace) plugin, I think I took Neovim's highlight `Whitespace` group as part of that plugin. I apologize for that and appreciate you reporting it.

**Reply by datwaft** [reply]
> Hi David! - in the last update where I added support for the [vim-better-whitespace](https://github.com/ntpeters/vim-better-whitespace) plugin, I think I took Neovim's highlight `Whitespace` group as part of that plugin. I apologize for that and appreciate you reporting it.

Don't worry. Thank you for the great colorscheme!

**Reply by wuelnerdotexe** [reply]
Thanks to you for choosing to be **Enfocado**. 👁
